scripts-available/CDB_Quota.sql

```diff
@@ -1,3 +1,22 @@
+CREATE OR REPLACE FUNCTION cartodb._CDB_total_relation_size(_schema_name TEXT, _table_name TEXT)
+RETURNS bigint AS
+$$
+BEGIN
+  IF EXISTS (
+      SELECT 1 FROM information_schema.tables
+      WHERE table_catalog = current_database()
+        AND table_schema = _schema_name
+        AND table_name = _table_name
+  )
+  THEN
+    RETURN pg_total_relation_size(format('"%s"."%s"', _schema_name, _table_name));
```

Comment:
Yes, although so is the other one, as it too is run within a single transaction block (all pl/pgsql functions are unitary transaction blocks, so state won't change on things in them unless you change that state yourself).